proxy/proxy.go

```diff
@@ -413,8 +413,6 @@ func copyStream(to flushedResponseWriter, from io.Reader, tracing *proxyTracing,
 	for {
 		l, rerr := from.Read(b)
 
-		tracing.logStreamEvent(span, StreamBodyEvent, fmt.Sprintf("%d", l))
```

Comment:
Could you please log the total number of copied bytes in a single log entry instead?

Reply:
done